documentation/guides/draft/use-cases/end-to-end-encryption-with-kafka/README.md

```diff
@@ -1,6 +1,136 @@
-# End-to-end encrypted messaging with Ockam and Kafka
+# End-to-end encrypted messaging with Kafka and Ockam Secure Channels
+
+## Background
+
+When using Apache Kafka and other stream processing services a common concern is data access and security.
+
+A lot of fields like finance and personal information handling require or advise data to be encrypted when stored and transferred.
```

Comment:
Many fields, such as finance and personal information handling, have statutory requirements for data to be encrypted when stored or in transit.